src/blenderbim/blenderbim/bim/module/material/operator.py

```diff
@@ -436,7 +444,7 @@ def execute(self, context):
                 new = props.material_set_attributes.add()
                 new.name = attribute.name()
                 new.is_null = material_set_data[attribute.name()] is None
-                new.string_value = "" if new.is_null else material_set_data[attribute.name()]
+                new.set_value("" if new.is_null else material_set_data[attribute.name()])
```

Comment:
This can use `import_attributes`

Reply:
This one looks like it is initialized differently but when testing with regular `import_attributes` I didn't see any difference